Python/pdf_reorder/README.md

```diff
@@ -0,0 +1,36 @@
+
+## A PDF reordering script
+
+This scrip can be used for reordering a pdf file which may have pages in wrong order.
+
+We used **pdfrw** library for this script. 
+
+A more efficient,faster alternative is [PyMuPDF](https://pypi.org/project/PyMuPDF/).       
+The PyMuPDF library can have dependency issues with versions of visual C++ hence we used a more simpler library pdfrw.
```

Comment:
Don't break lines too often. Doesn't look much standardized. Instead use complete sentences. Take a look at other Scripts and their corresponding README for a better idea.